src/main/java/ro/sapientia/furniture/model/Order.java

```diff
@@ -17,8 +21,12 @@ public class Order implements Serializable{
 	private static final long serialVersionUID = 1L;
 	
 	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator="pk_order_body")
```

Comment:
I think it would be better to name the generator "pk_order" and the others as well